packages/gestalt/src/Datapoint.js

```diff
@@ -0,0 +1,87 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PropTypes from 'prop-types';
+import Text from './Text.js';
+import Heading from './Heading.js';
+import Flex from './Flex.js';
+import IconButton from './IconButton.js';
+import Tooltip from './Tooltip.js';
+import DatapointTrend from './DatapointTrend.js';
+
+type TrendObject = {|
+  accessibilityLabel: string,
+  value: number,
+|};
+
+type Sentiment = 'good' | 'bad' | 'neutral' | 'auto';
+
+type Props = {|
+  infoText?: string,
+  size?: 'md' | 'lg',
+  title: string,
+  trend?: TrendObject,
+  trendSentiment?: 'good' | 'bad' | 'neutral' | 'auto',
+  value: string,
+|};
+
+export default function Datapoint({
+  infoText,
+  size = 'md',
+  title,
+  trend,
+  trendSentiment = 'auto',
+  value,
+}: Props): Node {
+  const valueSize = size === 'lg' ? 'lg' : 'sm';
+  const percentChangeGap = size === 'lg' ? 4 : 2;
+
+  return (
+    <Flex gap={1} direction="column">
+      <Flex gap={1} alignItems="center" minHeight={24}>
+        <Text size="sm">{title}</Text>
+        {infoText && (
+          <Tooltip text={infoText}>
```

Comment:
I think this needs an `idealDirection="up"` to prevent the Tooltip from covering the Datapoint content when possible

Reply:
Love it! Done!